app/response_processor.py

```diff
@@ -13,13 +13,24 @@ def connect_to_ftp():
     return ftp
 
 
-def get_ftp_folder(survey_response):
-    if 'heartbeat' in survey_response and survey_response['heartbeat'] is True:
+def get_ftp_folder(survey):
+    if 'heartbeat' in survey and survey['heartbeat'] is True:
         return settings.FTP_HEARTBEAT_FOLDER
     else:
         return settings.FTP_FOLDER
 
 
+def is_census_transform(survey):
```

Comment:
This method would be better named `is_census` - it's checking a survey to see if it is a census or not (we just so happen to then use that to determine what transform to do).